features/fout-meldingen.feature

```diff
@@ -0,0 +1,136 @@
+#language: nl
+Functionaliteit: Gebruiksvriendelijke foutmeldingen
+
+    Als developer van een consumer applicatie
+    Wil ik gebruiksvriendelijke en duidelijke foutmeldingen
+    Zodat ik deze één op één kan gebruiken in de UI van mijn applicatie
+
+Rule: minimaal één optionele parameter is opgegeven
+
+    Scenario: Er zijn geen parameters opgegeven
+        Als '/wozobjecten' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                                                                                                             |
+        | title  | Tenminste één parameter moet worden opgegeven.                                                                                                     |
+        | status | 400                                                                                                                                                |
+        | detail | Parameter 'rsin', 'kvkNummer', 'adresseerbaarObjectIdentificatie', 'nummeraanduidingIdentificatie' of 'postcode' en 'huisnummer' is niet opgegeven |
+        En bevat de response geen invalidParams
+
+Rule: alleen gespecificeerde parameters mogen worden opgegeven
+
+    Scenario: Niet gespecificeerd parameter
+        Als '/wozobjecten?identificatie=123456789' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                       |
+        | title  | Een of meerdere parameters zijn niet correct.                |
+        | status | 400                                                          |
+        | detail | Zoeken met parameter 'identificatie' wordt niet ondersteund. |
```

Comment:
Hier maak je deze melding specifiek voor "identificatie, maar er akn ook een andere parameter opgegeven worden die niet ondersteund wordt.

Reply:
Een scenario is een niet-uitputtend voorbeeld. De scenario is uitgebreid om te verduidelijken dat alleen WOZ-object kenmerken die zijn gespecificeerd als zoek parameter als zoek parameter kan worden gebruikt.

Reply:
Het gaat hier toch ook om specificatie van het gedrag van de API? En niet om het leveren van documentatie? Als dat de bedoeling is (documentatie) dan is het niet duidelijk genoeg, anders hadden we geen vragen gekregen van BAG en WOZ toch?